modules/navier_stokes/src/fvkernels/PINSFVEnergyAmbientConvection.C

```diff
@@ -40,10 +40,9 @@ PINSFVEnergyAmbientConvection::PINSFVEnergyAmbientConvection(const InputParamete
 ADReal
 PINSFVEnergyAmbientConvection::computeQpResidual()
 {
+  const auto elem = makeElemArg(_current_elem);
```

Comment:
```suggestion
  const auto & elem = makeElemArg(_current_elem);
```
please get all other occurences